assets/dev/js/editor/editor-base.js

```diff
@@ -977,7 +977,7 @@ export default class EditorBase extends Marionette.Application {
 		// TODO: Should be command?
 		jQuery( '#elementor-preview-loading' ).show();
 
-		this.$preview[ 0 ].contentWindow.location.reload( true );
+		this.$preview[ 0 ].src = this.config.initial_document.urls.preview;
```

Comment:
Please check the case when you are editing loop item that the initial document was some page and then switch the canvas view

Reply:
NO